src/towncrier/_builder.py

```diff
@@ -176,14 +184,13 @@ def split_fragments(
             if definitions[category]["showcontent"] is False:
                 content = ""
 
-            texts = section.get(category, OrderedDict())
+            texts = section.setdefault(category, OrderedDict())
 
-            if texts.get(content):
-                texts[content] = sorted(texts[content] + [ticket])
-            else:
-                texts[content] = [ticket]
-
-            section[category] = texts
+            tickets = texts.setdefault(content, [])
+            if ticket:
+                # Don't add the ticket if it's blank (meaning it's an unlinked fragment).
```

Comment:
mabye something like this

the `if ticket` and then a comment about `if not ticket` is a bit confusing.
 
```suggestion
            if not ticket:
                """
                Don't add the ticket if we don't have one.
                This can happen for unlinked / orphan fragments.
                """
                continue
```

Reply:
Clarified.